src/App.tsx

```diff
@@ -1,61 +1,142 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoWithUser } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+export const firstTodos = todosFromServer.map(todo => ({
```

Comment:
The name "first todos" is a bit confusing, it might be better to call it "initial todos" so that it is more clear what kind of information it is.